### PR TITLE
Features/design

### PR DESCRIPTION
**Pull request by domicode**
- [x] first version of design

### CODE REVIEW

File: assets/stylesheets/global.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+         url('equipo-squaredance-2-webfont.woff') format('woff'),
+         url('equipo-squaredance-2-webfont.ttf') format('truetype'),
+         url('equipo-squaredance-2-webfont.svg#equipo_squaredanceregular') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'equipo_squaredancelight';
+    src: url('equipo-quaredance-2-light-webfont.eot');
```
Comment (huerlisi) [inline assets/stylesheets/global.scss]: I think you lost an `s` here: `equipo-quaredance` :-)

**Comment by huerlisi** [comment]
:+1: go for it!

**Comment by koffeinfrei** [comment]
- Hmm the glyphs have horizontal lines, I don't think that's how it's supposed to be?
- What's the idea with the title "panter B dashhboard"? a) that should be the same font, b) it's somewhat unnecessary

**Comment by domicode** [comment]
Can you make a screenshot? I don't know what you mean by horizontal lines. 

a) sorry, my bad, didn't test it in firefox. Fixed it. 
b) yes but it looks cooler (in my opinion) ;) 
dashboard_w_header
dashboard_w_o_header

**Comment by koffeinfrei** [comment]
Aaah now that's a whole different story, for me it looks like this:

panter ag 2015-07-10 14-50-42

BTW there's an "h" too many in the title.

**Comment by domicode** [comment]
yeah like that it makes not much sense and it looks bad. Fixed that problem for firefox/globally. 
About the lines, I get now what you mean. But don't really know why the font rendering is doing that, will have to check what could be the problem there.

**Pull request merged**